data-catalog-backend-app/src/main/java/no/nav/data/catalog/backend/app/informationtype/InformationTypeESDocumentResponse.java

```diff
@@ -0,0 +1,67 @@
+package no.nav.data.catalog.backend.app.informationtype;
+
+import no.nav.data.catalog.backend.app.codelist.ListName;
+import no.nav.data.catalog.backend.app.policy.Policy;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static no.nav.data.catalog.backend.app.codelist.CodelistService.codelists;
+
+public class InformationTypeESDocumentResponse {
+    private String elasticsearchId;
+    private Long informationTypeId;
+    private String name;
+    private String description;
+    private Map category;
+    private List<Map> producer;
+    private Map system;
+    private Boolean personalData;
+    private List<Policy> policies;
+
+    InformationTypeESDocumentResponse(InformationTypeESDocument informationTypeESDocument) {
+        this.elasticsearchId = informationTypeESDocument.getInformationType().getElasticsearchId();
```

Comment:
this.elasticsearchId = informationTypeESDocument.getInformationTypeResponse().getElasticsearchId()
...
this.producer = informationTypeESDocument.getProducer()   // la dette gjøres i InformationTypeResponse og fjern metodene her